frontend/src/components/MapMenu/MapMenu.test.tsx

```diff
@@ -10,8 +10,9 @@ beforeEach(() => {
     mockMatchMedia();
 });
 
-test("Filter menu is opened when pressing filter button and closed when pressing confirm", async () => {
+test("Filter menu is opened when pressing filter button and closed when pressing confirm", async () => {  
     const onSearchClicked = jest.fn();
+    jest.setTimeout(10000);
```

Comment:
What is this doing here?

Reply:
I was running 'npm run test:coverage', and my update made this test failing. Error message was: "Exceeded timeout of 5000 ms for a test." with suggestion "Use jest.timeout(newTimeout) to increase the timeout value, if this is a long-running test."

Well, it might be, that it failed just locally. So, if you suggest, I may try to commit without this line although it's is failing in my local environment (generally my laptop has been slow).

Reply:
Done